Example install files for Elasticsearch and Python 3.

rsdoiel opened the pull request:
Anticipating that many of our projects may take advantage of modern Python or Elasticsearch I've included two example installation instructions. If those tools were requirements then they could be linked into README.md or INSTALL.md or if not needed removed.

rsdoiel committed:
This is an example set of install instructions

If your project uses Elasticsearch you can use these instructions to describe its installation process.

rsdoiel committed:
Example installation instructions

If your project depends on Python 3.x you can use these examples for installing Python3 using Debian/Ubuntu or MacPorts package managers.

mhucka reviewed (changes requested):
See comments on files, below.

mhucka commented on INSTALL-Elasticsearch.md:
```diff
@@ -0,0 +1,96 @@
+
+    https://www.elastic.co/start
+
+Elasticsearch needs to be running locally to use Observatory. Observatory
```
Observatory

mhucka commented on INSTALL-Python3.md:
```diff
@@ -0,0 +1,53 @@
+
+It is possible to use your default OS provided python if it is version
+>= 3.6. Many distributions still ship with version 2.7 out of the box
+and pipy's pip corresponding to version 2.7. Observatory REQUIRES 
```
Observatory

mhucka reviewed (changes requested):
This references Observatory, which is unfortunately not suitable for a general-purpose template. Could this text be changed to be more generic?

rsdoiel committed:
removed observatory references, made shorter

rsdoiel committed:
changed https to http for localhost link.

rsdoiel committed:
removed /_status path for localhost url

rsdoiel committed:
Added some notes about managing Elasticsearch

Add notes to mange Elasticsearch on Debian/Ubuntu and via MacPorts.

rsdoiel committed:
Copyediting

Cleaned up copy, removed observatory reference, added example description of invoking the right pip using `python3 -m pip` command idiom.

rsdoiel commented:
Mike, I've updated the copy for both the Python and Elasticsearch. Let me know if you want additional changes. Thanks, Robert
